pkg/validation/spec/response.go

```diff
@@ -51,7 +51,11 @@ func (r *Response) UnmarshalJSON(data []byte) error {
 	if err := json.Unmarshal(data, &r.Refable); err != nil {
 		return err
 	}
-	return json.Unmarshal(data, &r.VendorExtensible)
+	if err := json.Unmarshal(data, &r.VendorExtensible); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
is this change a no-op?

Reply:
yes, was a formatting change for consistency